docs/content/guides/dagster/ml-ops.mdx

```diff
@@ -0,0 +1,221 @@
+from sklearn.model_selection import train_test_split
+from dagster import multi_asset, AssetOut
+
+@multi_asset(outs={'training_data': AssetOut(), 'test_data': AssetOut()})
+def training_test_data(hackernews_stories):
+    hackernews_stories = hackernews_stories
+    X = hackernews_stories.title
+    y = hackernews_stories.descendants
+    """Split the dataset to reserve 20% of records as the test set"""
+    X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2)
+    return (X_train,  y_train), (X_test, y_test)
+```
+
+Next, we will take both the training and test data subsets and [tokenize the titles](https://en.wikipedia.org/wiki/Lexical_analysis) e.g. take the words and turn them into columns with the frequency of terms for each records to create [features](https://en.wikipedia.org/wiki/Feature_\(machine_learning\)) for the data. To do this, we will be using the training set to fit the tokenizer, in this case we are using [TfidfVectorizer](https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.TfidfVectorizer.html) and then transforming both the training and test set based with that tokenizer.
+
+```python file=/guides/dagster/ml_ops/ml_ops.py startafter=vectorizer_start endbefore=vectorizer_end
+from sklearn.feature_extraction.text import TfidfVectorizer
+import numpy as np
+
+@multi_asset(outs={'Tfidf_Vectorizer': AssetOut(), 'transformed_training_data': AssetOut()})
+def transformed_train(training_data):
+    X_train,  y_train = training_data
+    """Initiate and fit the tokenizer on the training data and transform the training dataset"""
+    vectorizer = TfidfVectorizer()
+    transformed_X_train = vectorizer.fit_transform(X_train)
+    transformed_X_train = transformed_X_train.toarray()
+    y_train = y_train.fillna(0)
+    transformed_y_train = np.array(y_train)
+    return vectorizer, (transformed_X_train, transformed_y_train)
+
+@asset
+def transformed_test_data(test_data, Tfidf_Vectorizer):
+    X_test, y_test = test_data
+    """Use the fitted tokenizer to transform the test dataset"""
+    transformed_X_test = Tfidf_Vectorizer.transform(X_test)
+    transformed_y_test = np.array(y_test)
+    y_test = y_test.fillna(0)
+    transformed_y_test = np.array(y_test)
+    return transformed_X_test, transformed_y_test
+```
+
+We have also transformed the dataframes into numpy arrays and removed na values to prepare the data for training.
+
+### Model training
+
+At this point, we have X_train, y_train, X_test, y_test ready to go for our model. We can use any number of models to train our model from libraries like [sklearn](https://scikit-learn.org/), [tensorflow](https://www.tensorflow.org/) and [pytorch](https://pytorch.org/).
+
+In our example, we will be training a [XGBoost model](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBRegressor) to predict a numerical value.
```

Comment:
```suggestion
In our example, we will train an [XGBoost model](https://xgboost.readthedocs.io/en/stable/python/python_api.html#xgboost.XGBRegressor) to predict a numerical value.
```

🔧 Suggesting we use `we will train` instead of `we will be training` - it's less wordy and more action-forward.